tree/dataframe/inc/ROOT/RCsvDS.hxx

```diff
@@ -75,7 +77,8 @@ protected:
    std::string AsString();
 
 public:
-   RCsvDS(std::string_view fileName, bool readHeaders = true, char delimiter = ',', Long64_t linesChunkSize = -1LL);
+   RCsvDS(std::string_view fileName, bool readHeaders = true, char delimiter = ',', Long64_t linesChunkSize = -1LL,
+          std::string fillValue = "NaN", unsigned int lookDownForType = 10U);
```

Comment:
About `lookDownForType`: I'm not sure we should expose a parameter for this purpose or just have an internal behaviour, does the same exist e.g. in the pandas CSV reader?

Reply:
I just answered on how other csv readers work in another comment, but it also makes sense to add that here: 

Pandas puts NaN everywhere it sees an empty cell. And it produces vectors of objects, i.e. a column might have integer and a string inside it.

Numpy accepts only numerical values and represent them as floats. On the place of all other input cells (strings, chars, or empty cell) is nan.

I have no strong opinion for or against keeping this parameter. Let me know what is your view, given the behaviour of other csv readers.

Reply:
So if I understand correctly this parameter does not exist in other tools, they just internally solve the typing internally. Then my opinion would be not to expose such a parameter. At least for now, we can always add it later.